src/Umbraco.Web/Editors/PasswordChanger.cs

```diff
@@ -9,6 +9,7 @@
 using Umbraco.Web.Security;
 using IUser = Umbraco.Core.Models.Membership.IUser;
 
+//Migrated to .NET CORE
```

Comment:
Can this be deleted

Reply:
Nope, not yet.. Still used in members implementation